test/unit/effectsTest.js

```diff
@@ -75,7 +75,7 @@ test('random background effect', async t => {
   nativeAPI.p5_.randomSeed(0);
 
   nativeAPI.setBackgroundEffect('rand');
-  t.equal(nativeAPI.world.bg_effect, 'fireworks');
```

Comment:
Why does this change?

Reply:
My understanding is that this changes because we seed randomness to return the same value everytime and adding additional values to the list changes the values in the randomness calculation.

Reply:
ah makes sense, thanks